packages/jest-core/src/runJest.ts

```diff
@@ -253,9 +337,10 @@ export default (async function runJest({
     await runGlobalHook({allTests, globalConfig, moduleName: 'globalTeardown'});
   }
 
-  return processResults(results, {
+  return await processResults(results, {
```

Comment:
?

Reply:
It's now async, since for JSON lines it'll now wait on stdout for buffering output.

Reply:
This whole function is async already, `await`ing here just creates an extra promise instead of returning the promise from `processResults`